scripts/tools/tools-object.js

```diff
@@ -0,0 +1,57 @@
+const schema = require("./tools-schema.json");
+const axios = require('axios')
+var validate = require("jsonschema").validate;
```

Comment:
You should use `ajv` https://github.com/ajv-validator/ajv - it's better tool to validate JSON Schema

Reply:
I tried with `ajv` library but weirdly, it is passing the wrong JSON data also and hence, breaking my functionality at `getContent` function. Here's the wrong JSON data file:


<details>
<summary> Wrong format of JSON according to schema </summary>

```JSON
{
    "title":"asyncapi_gencpp",
    "description": "C++ code generator for serializing and deserializing Async API components and messages based on a specification file.",
    "maintainers":[
        "marcalban", "pjreed"
    ],
    "links":{
        "websiteUrl": "https://github.com/hatchbed/asyncapi_gencpp",
    },
    "filters": {
        "language": "C++",
        "technology": ["Python"],
        "category": ["generator"],
        "isCommercial": false
    }
}
```

</details>

the `category` field inside `filters` should be `categories` but it is still evaluated as correct by validate function of `ajv`.

Reply:
leave this, I solved this error and pushed the updated changes.